server/stt/stt_events_ml.py

```diff
@@ -217,6 +217,7 @@ def set_location_details(self, item, location_xml, notes):
         try:
             name = location_xml.find(self.qname("name")).text
             location["name"] = name
+            location["unique_name"] = name
```

Comment:
not sure if that would be unique actually, wouldn't it require also city and country or something?

Reply:
you're right name alone isn't unique. Let's include city and country.